pinc/bad_bytes.inc

```diff
@@ -898,30 +897,22 @@ function get_remarks_for_bad_byte_sequence($raw)
 
         $why_bad = "HTML numeric character reference";
     }
-    elseif(utf8_strlen($raw) == 1 && is_utf8($raw))
-    {
-        $why_bad = "Codepoint not valid for project";
-        $intended_character = "";
-    }
     else
     {
         list($code_point, $why_bad) = $_bad_byte_sequences[$raw];
     }
 
-    if(!isset($intended_character))
+    @$info = $_character_data[$code_point];
```

Comment:
`if(isset($_character_data[$code_point]))` would be clearer than using `@`.

Reply:
I agree. Found the null coalescing operator.